test/fixtures/ai/post_ai_text_gen_response.json

```diff
@@ -0,0 +1,5 @@
+{
+    "answer": "The document is currently in review and awaiting approval.",
+    "created_at": "2024-07-09T11:29:46.835Z",
+    "completion_reason": "done"
+}
```

Comment:
The `created_at` field have a different structure in generated SDK (it's an object with value field) - https://github.com/box/boxcli/pull/538/files#diff-d8df5e635c0ea2eede968c858e24c4a82154fcb07e0b4907032c3ec70b987435. Do we want to wrap it here/unwrap when using generated SDK to avoid breaking changes?

Reply:
@congminh1254 any comment on this one?